src/metabase/models/params.clj

```diff
@@ -67,6 +68,28 @@
     :else
     field-id-or-form))
 
+(def ^:dynamic *ignore-current-user-perms-and-return-all-field-values*
+  "Whether to ignore permissions for the current User and return *all* FieldValues for the Fields being parameterized by
+  Cards and Dashboards. This determines how `:param_values` gets hydrated for Card and Dashboard. Normally, this is
+  `false`, but the public and embed versions of the API endpoints can bind this to `true` to bypass normal perms
+  checks (since there is no current User) and get *all* values."
+  false)
+
+(defn- field-ids->param-field-values-ignoring-current-user
```

Comment:
This doesn't create them if they don't exist. There are some functions in `metabase.models.params.field-values` that handle all of the logic. Do we need to bind current-perms to admin level and call those functions here?